### PR TITLE
Fix ESLint NavBar.js

### PR DESCRIPTION
**Pull request by sarovin**

### CODE REVIEW

**Comment by sarovin** [comment]
4 eslint errors remaining but i need help.

File: src/NavBar.js [hunk 1]
```diff
@@ -194,15 +45,15 @@ const styles = StyleSheet.create({
     fontWeight: '500',
     color: '#0A0A0A',
     position: 'absolute',
-    top: Platform.OS === 'ios' ? 20 : 0,
+    top: Platform.OS === 'ios' || Platform.Version > 19 ? 20 : 0,
```
Comment (charpeni) [inline src/NavBar.js]: Shouldn't be there.

File: src/NavBar.js [hunk 2]
```diff
@@ -194,15 +45,15 @@ const styles = StyleSheet.create({
...
     left: 0,
     right: 0,
   },
   header: {
     backgroundColor: '#EFEFF2',
     paddingTop: 0,
     top: 0,
-    height: Platform.OS === 'ios' ? 64 : 44,
+    height: Platform.OS === 'ios' || Platform.Version > 19 ? 64 : 44,
```
Comment (charpeni) [inline src/NavBar.js]: Same as line 48, shouldn't be there.

**Comment by charpeni** [comment]
I will take a look at remaining errors.

File: src/NavBar.js [hunk 3]
```diff
@@ -256,5 +107,285 @@ const styles = StyleSheet.create({
...
+          />
+        );
+      }
+      if (state.onRight && (state.rightTitle || state.rightButtonImage)) {
+        return (
+          <TouchableOpacity
+            key={'rightNavBarBtn'}
+            testID="rightNavButton"
+            style={[styles.rightButton, state.rightButtonStyle]}
+            onPress={state.onRight.bind(null, state)}
```
Comment (charpeni) [inline src/NavBar.js]:
```javascript
onPress={onPress}
```

And defined `onPress` before the current return : 
```javascript
const onPress = state.onRight.bind(null, state);
```

File: src/NavBar.js [hunk 4]
```diff
@@ -256,5 +107,285 @@ const styles = StyleSheet.create({
...
+        }
+      }
+
+      if (onPress && (state.leftTitle || buttonImage)) {
+        return (
+          <TouchableOpacity
+            key={'leftNavBarBtn'}
+            testID="leftNavButton"
+            style={[styles.leftButton, state.leftButtonStyle]}
+            onPress={onPress.bind(null, state)}
```
Comment (charpeni) [inline src/NavBar.js]:
```javascript
onPress={onPress}
```

And defined `onPress` before the current return : 
```javascript
onPress = onPress.bind(null, state);
```

**Comment by charpeni** [comment]
I just added some comments about remaining errors, feel free to comment.

**Comment by sarovin** [comment]
`'onPress' is already declared in the upper scope.`
I have changed OnPress in OnPressBind, it's ok?

File: src/NavBar.js [hunk 5]
```diff
@@ -277,12 +278,13 @@ class NavBar extends React.Component {
       }
 
       if (onPress && (state.leftTitle || buttonImage)) {
+        const onPressBind = onPress.bind(null, state);
```
Comment (charpeni) [inline src/NavBar.js]: Reassign `onPress` instead of a declaration and it should be fine.
```javascript
onPress = onPress.bind(null, state);
```

**Comment by charpeni** [comment]
LGTM. Good work @sarovin, thank you! :)

**Pull request merged**